keyboards/iris/keymaps/fate/config.h

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2017 Danny Nguyen <danny@keeb.io>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace the boilerplate with just `#pragma once`. 

Specifically, replace:
```h
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

#include "config_common.h"
``` 

And the `#endif` at the end with `#pragma once`